pkg/storage/storage.go

```diff
@@ -187,13 +190,29 @@ type ChangelogBackend interface {
 	ReadChanges(ctx context.Context, store, objectType string, paginationOptions PaginationOptions, horizonOffset time.Duration) ([]*openfgapb.TupleChange, []byte, error)
 }
 
+// CurrentTime has functionality of Now that allows mocking of current time
+type CurrentTime interface {
```

Comment:
nit pick - `TimeProvider` I think is a better interface name here.

See https://go.dev/doc/effective_go#interface-names